README.md

```diff
@@ -45,7 +45,20 @@ The `ntripping` utility has the following usage:
             --url <url>             [default: na.skylark.swiftnav.com:2101/CRS]
 
 Different resources can be requested from different locations. By default, a San
-Francisco latitude, longitude, and height will be used.
+Francisco latitude, longitude, and height will be used. The data returned is in RTCM format.
```

Comment:
> The data returned is in RTCM format.

NTRIP can return other protocols, like BINEX. So this isn't true.

Reply:
Is there a way to specify the protocol you want returned? It wasn't obvious what the default format was, or how to get that information.

Reply:
You can read up the NTRIP protocol - it goes through how information that a NTRIP caster provides and in what format a source provides through a source table. There is no default format and there is no way to specify the format you want returned - it's part of the source information captured in the source table. This tool is definitely expecting some familiarity around using NTRIP - we shouldn't repeat general NTRIP documentation here.

Reply:
Added a link to the NTRIP protocol and removed this